src/client/Microsoft.Identity.Client/Kerberos/KerberosSupplementalTicketManager.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Json;
+using Microsoft.Identity.Json.Linq;
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.Identity.Client.Kerberos
+{
+    /// <summary>
+    /// Helper class to manage Kerberos Ticket Claims.
+    /// </summary>
+    public static class KerberosSupplementalTicketManager
+    {
+        private const int DefaultLogonId = 0;
+        private const string KerberosClaimType = "xms_as_rep";

```

Comment:
We try to follow the pattern of _(lowercase char)****
for example "_defaultLogonId"

Reply:
Changed.